docs/src/components/programs_and_authorizations.md

```diff
@@ -0,0 +1,40 @@
+# Programs and Authorizations
+
+A **Valence Program** is an instance of the **Valence Protocol**. It is a particular arrangement and configuration of [accounts](./accounts.md) and [libraries](./libraries_and_functions.md) across multiple [domains](./domains.md) e.g. a POL (protocol-owned liquidity) lending relationship between two parties. Similarly to how a _library_ exposes executable _functions_, programs are associated with a set of executable **Subroutines**.
+
+A **Subroutine** is a vector of **Functions**. A **Subroutine** can call out to one or more **function(s)** (from a single library, or from different libraries), on one or more (execution) domain(s). 
+
+A **Subroutine** can be:
+
+- **Sequential**, i.e., execute function one, if it succeeds execute function two, and if that succeeds do function three, and so on.
```

Comment:
We use the term non atomic. As both Atomic and NonAtomic are sequential in nature, the only difference is that either all get executed or none